doc-Community_Collaboration_Guide/topics/creating_a_new_repository.adoc

```diff
@@ -1,11 +1,153 @@
+[[ccg-create-repo]]
 == Creating a New Repository
 
 Placeholder text.
 
+[[ccg-create-repo-setup]]
 === Setting up the Repository
 
 Placeholder text.
 
+[[ccg-create-repo-synch]]
 === Configuring Content Synchronization
 
 Placeholder text.
+
+[[ccg-create-repo-synch-methods]]
+==== Synchronization Methods
+
+The following synchronization methods are currently recommended:
+
+* <<ccg-create-repo-synch-batchcommit,Batch Commit>>
+
+
+[[ccg-create-repo-synch-batchcommit]]
+==== Batch Commit Synchronization Method
+
+[[ccg-create-repo-synch-batchcommit-when]]
+===== When to Use this Approach
+
+The batch commit process is a good approach to use when you need to incorporate upstream content into product documentation and the product contains a good deal of additional product-specific content.
```

Comment:
The/A batch commit process

Reply:
In this case, I think it's "The". Because "The" batch process is a particular example of "a" synchronization method.